### PR TITLE
Link prerequisites from home page

### PR DESCRIPTION
**Pull request by nyechiel**
Many users are struggling with finding our requirements; hopefully this will make it a bit easier for them.

Signed-off-by: nyechiel <nyechiel@redhat.com>

### CODE REVIEW

File: src/content/_index.en.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
 * [`subctl`](./operations/deployment/), a friendly deployment tool
 * Support for interconnecting clusters with [overlapping CIDRs](./getting_started/architecture/globalnet/)
 
+{{% notice note %}}
+A few requirements need to be met before you can begin. Check the [Prerequisites](./getting_started/#prerequisites) section for more
```
Comment (sridhargaddam) [inline src/content/_index.en.md:22]: Nir, i think we currently do not have a prerequisite page on the website. Did you miss including the prerequisite as part of this PR? Or are you planning to push a separate PR for that?

Comment (nyechiel) [reply src/content/_index.en.md:22]: Yes, we do. Check the link ;)

Comment (sridhargaddam) [reply src/content/_index.en.md:22]: Oh yes, you are right #prerequisites is a link in getting_started. I was thinking its a separate page on its own. Thanks.

**Review by sridhargaddam — APPROVED** [review]

**Review by aswinsuryan — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**